doc/user/content/transform-data/json.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "JSON to SQL"
+description: "Create a view from sample JSON data."
+menu:
+  main:
+    parent: 'transform'
+    name: JSON
+    weight: 20
+---
+
+Working with JSON formatted data in SQL can be tedious, because you need to constantly deconstruct
+the JSON blob. The below tool accepts a JSON sample and will generate a SQL view with the
+individual fields mapped to columns.
+
+
+<div class="json_widget">
+    <span class="input_container">
+        <span class="input_container-text">
+            <input id="view_name" placeholder="View Name">
+            <input id="source_name" placeholder="Relation Name">
+            <input id="column_name" placeholder="JSON Column Name">
+        </span>
+    <fieldset class="input_container-radio">
+        <legend>Kind of SQL Object</legend>
+        <span>
+            <input type="radio" id="view" name="type_view" value="view"/>
+            <label for="view">View</label>
+        </span>
+        <span>
+            <input type="radio" id="materialized-view" name="type_view" value="materialized-view"/>
+            <label for="materialized-view">Materialized View</label>
+        </span>
+    </fieldset>
+    </span>
+    <div class="json">
+        <textarea id="json_sample" placeholder="JSON Sample"></textarea>
+        <div id="error_span" class="error">
+            <p id="error_text"></p>
+        </div>
+    </div>
+    <pre class="sql_output">
+        <code id="output" class="sql_output-code"></code>
+    </pre>
+</div>
+
+<script>
+
+/* Helper Methods
+
+If this wasn't a simple script these would be in a `utils.js` or come from lodash.
+*/
+
+function escapeString(s) {
+    return s.replace(`'`, `''`);
+}
+
+function escapeIdent(s) {
+    return s.replace(`"`, `""`);
+}
+
+function clone(x) {
+    return JSON.parse(JSON.stringify(x))
+}
+
+function debounce(callback, wait) {
+    let timeout;
+    return (...args) => {
+        const context = this;
+        clearTimeout(timeout);
+        timeout = setTimeout(() => callback.apply(context, args), wait);
+    }
+}
+
+/* JSON Parsing and SQL conversion */
+
+const error_span = $("#error_span");
+const error_text = $("#error_text");
```

Comment:
Variables names are typically camelCase in javascript.

Reply:
Ahh! This kept biting me haha, some things were camel case but not all, just changed everything to be camel